functional/debug_console/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SCRIPT_PATH=$(dirname "$0")
+
+source "${SCRIPT_PATH}/../../lib/common.bash"
+
+extract_kata_env
+if [ -z "${INITRD_PATH}" ]; then
+	info "Skip: initrd path is empty"
+	exit 0
+fi
+
+
+info "Enable debug console"
+sudo cp "${RUNTIME_CONFIG_PATH}" "${RUNTIME_CONFIG_PATH}.old"
+
+info "Disable debug"
+sudo sed -i \
+	 -e 's/^#*\(kernel_params\).*=.*$/\1 = \"agent.debug_console\"/g' "${RUNTIME_CONFIG_PATH}" \
+	 -e 's/^\(enable_debug.*\)/#\1/g' \
```

Comment:
Do you need to disable *all* the debug options, or can you just do `crudini --set $file proxy.kata enable_debug`?